internal/sshd/client.go

```diff
@@ -47,6 +47,7 @@ func (c *Client) handleHint(req *ssh.Request) {
 
 func (c *Client) handleTCPIPForward(
 	ctx context.Context,
+	proxy conf.Proxy,
 	cancel context.CancelFunc,
```

Comment:
`ctx` and `cancel` are a pair, so let's put `proxy` in the third place of arguments.